### PR TITLE
Bug 1554177 Added note box to mtu topics

### PR DESCRIPTION
**Pull request by bfallonf**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1554177

Problems making it build. Maybe because of the modularisation? @vikram-redhat will look at it.

### CODE REVIEW

**Review by knobunc — APPROVED** [review]
LGTM.  Thanks Brice

**Comment by bfallonf** [comment]
Updated to copy the note box between files instead of using includes. Turns out you can't have includes inside of includes and expect them to build correctly...

@openshift/team-documentation PTAL

File: day_two_guide/topics/verifying_mtu.adoc [hunk 1]
```diff
@@ -125,8 +125,9 @@ networkConfig:
 +
 [NOTE]
 ====
-Changing the MTU size must be performed on all masters and nodes due to being
-part of the OCP SDN.
+You must change the MTU size on all masters and nodes that are part of the
+{product-title} SDN. Also, the MTU size of the tun0 interface must be the same
+across all each nodes that are part of the cluster.
```
Comment (mburke5678) [inline day_two_guide/topics/verifying_mtu.adoc]: Fix the _all each_. Don't need both! ;-)

File: install_config/configuring_sdn.adoc [hunk 2]
```diff
@@ -183,6 +183,13 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
+[NOTE]
+====
+You must change the MTU size on all masters and nodes that are part of the
+{product-title} SDN. Also, the MTU size of the tun0 interface must be the same
+across all each nodes that are part of the cluster.
```
Comment (mburke5678) [inline install_config/configuring_sdn.adoc]: Fix the _all each_.

File: scaling_performance/network_optimization.adoc [hunk 3]
```diff
@@ -89,6 +89,13 @@ networkConfig:
 `redhat/openshift-ovs-networkpolicy` for the *ovs-networkpolicy* plug-in. This
 can also be set to any other CNI-compatible plug-in as well.
 
+[NOTE]
+====
+You must change the MTU size on all masters and nodes that are part of the
+{product-title} SDN. Also, the MTU size of the tun0 interface must be the same
+across all each nodes that are part of the cluster.
```
Comment (mburke5678) [inline scaling_performance/network_optimization.adoc]: Fix the _all each_.

**Comment by mburke5678** [comment]
@bfallonf One tiny nit x3. Otherwise LGTM

**Pull request merged**